ui/src/authors/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -144,38 +142,28 @@ function DetailPage({
         </ContentBox>
       ),
     },
-  ];
-
-  if (canAccessDataTab) {
-    tabItems = [
-      ...tabItems,
-      {
-        label: (
-          <Tooltip title="Datasets from the author">
-            <span>
-              <span>Datasets</span>
-            </span>
-          </Tooltip>
-        ),
-        key: '2',
-        children: (
-          <ContentBox className="remove-top-border-of-card">
-            <AuthorDataContainer />
-          </ContentBox>
-        ),
-      },
-    ];
-  }
-
-  tabItems = [
-    ...tabItems,
+    {
```

Comment:
we never tested the `canAccessDataTab`?

Reply:
you mean by actual ui tests ?

Reply:
yep, i'm just asking in case we have to remove them too

Reply:
Nope, we actually don't have tests for this DetailPageContainer (and for others too, like Seminars, Institutions etc). It would be good to spend some time and add them at some point. Maybe we can tackle it while we migrate the enzyme to rtl.